### ISSUE TITLE
Import problems after upgrading to version 0.3.8

### ISSUE THREAD

**Issue by tekrei** [usnistgov/jsfive#30]
I am using jsfive in a React project created with CRA.

- react version is `^17.0` (`17.0.2` is installed)
- react-scripts version is `^4.0` (`4.0.3` is installed)

Everything was working fine until I upgraded **jsfive** to latest **0.3.8** version. I am currently having problems when using the library.

Here is what I have tried so far.

I was using `import * as hdf5 from "jsfive/dist/index";` before and here is the compile error I receive after update (using `index.js`, using without `index` are also giving the same error):

```
./node_modules/jsfive/dist/index.js 5186:14
Module parse failed: Unexpected token (5186:14)
File was processed with these loaders:
 * ./node_modules/react-scripts/node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| };
| var BTreeV1 = class extends AbstractBTree {
>   B_LINK_NODE = /* @__PURE__ */new Map([["signature", "4s"], ["node_type", "B"], ["node_level", "B"], ["entries_used", "H"], ["left_sibling", "Q"], ["right_sibling", "Q"]]);
|
|   _read_node_header(offset, node_level) {
```

I tried using `import * as hdf5 from "jsfive";` as suggested in the latest release (and README), no compile error but opening a file gives the following error in the console:

```
Uncaught TypeError: jsfive__WEBPACK_IMPORTED_MODULE_1__.File is not a constructor
    onloadend HDF5File.js:66
```

Finally I try `const hdf5 = import("jsfive");` as suggested in README (but without await), I am receiving the following in the console:

```
Uncaught TypeError: hdf5.File is not a constructor
    onloadend HDF5File.js:68
    HDF5File HDF5File.js:67
    React 5
    unstable_runWithPriority scheduler.development.js:468
    React 8
    unstable_runWithPriority scheduler.development.js:468
    React 15
    js index.js:7
    js main.chunk.js:3022
    Webpack 7
```

How can I import and use correctly?

**package.json**

```
{
  "name": "name",
  "description": "description",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "start": "CHOKIDAR_USEPOLLING=1 react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "dependencies": {
    "@fortawesome/fontawesome-svg-core": "^1.2",
    "@fortawesome/free-solid-svg-icons": "^5.15",
    "@fortawesome/react-fontawesome": "^0.1",
    "axios": "^0.25",
    "jsfive": "^0.3",
    "papaparse": "^5.3",
    "react": "^17.0",
    "react-alert": "^7.0",
    "react-alert-template-basic": "^1.0",
    "react-dom": "^17.0",
    "react-scripts": "4.0",
    "react-table": "^7.7",
    "web-vitals": "^2.1"
  },
  "devDependencies": {
    "@testing-library/jest-dom": "^5.16",
    "@testing-library/react": "^12.1",
    "@testing-library/user-event": "^13.5"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}


```

**Comment by bmaranville** [comment]
Some of these issues were found in #21, also - but I think they were resolved if using CRA version 5.0 instead of 4.0

I was able to compile and successfully run this test app with CRA:
```javascript
/* src/App.js */
import logo from './logo.svg';
import './App.css';

import * as hdf5 from 'jsfive';

async function test() {
  let response = await fetch("https://ncnr.nist.gov/pub/ncnrdata/vsans/202003/24845/data/sans59510.nxs.ngv");
  let ab = await response.arrayBuffer();
  var f = new hdf5.File(ab);
  console.log(f.keys); // result = [ "entry" ]
}
test();


function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default App;
```

here is the package.json from that test:
```json
{
  "name": "test",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@testing-library/jest-dom": "^5.16.2",
    "@testing-library/react": "^12.1.2",
    "@testing-library/user-event": "^13.5.0",
    "jsfive": "^0.3.8",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "5.0.0",
    "web-vitals": "^2.1.4"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```

**Comment by tekrei** [comment]
Thank you @bmaranville!

I didn't upgrade react-scripts to v5.0 yet because of a hot loading problem. It is working with v5.0 by using `import * as hdf5 from 'jsfive';`.
